data/src/main/scala/ch.epfl.scala.index.data/bintray/BintrayMeta.scala

```diff
@@ -32,13 +30,4 @@ object BintrayMeta extends BintrayProtocol {
       .sortBy(_.created)(Descending)
   }
 
-  def append(paths: DataPaths, meta: BintraySearch): Unit = {
```

Comment:
This is where it was refactored. I forgot to remove the above `append` method in the refactor.